### PR TITLE
DCOS-21229: Sorting in pods table

### PR DESCRIPTION
**Pull request by DanielMSchmidt**
This PR aims to write integration tests for sorting pod instances and add the functionality which seems to be missing

**Checklist**
- [x] Did you add a JIRA issue in a commit message or as part of the branch name?
- [ ] Did you add new unit tests?
- [x] Did you add new integration tests?
- [x] If this is a regression, did you write a test to catch this in the future?

### CODE REVIEW

File: plugins/services/src/js/containers/pod-instances/PodInstancesTable.js [hunk 1]
```diff
@@ -138,6 +138,8 @@ class PodInstancesTable extends React.Component {
   }
 
   getColumns() {
+    // TODO: add sort function here
```
Comment (weblancaster) [inline plugins/services/src/js/containers/pod-instances/PodInstancesTable.js]: @DanielMSchmidt the sorting methods shouldn't be written in this method. You can follow what we have at `https://github.com/dcos/dcos-ui/blob/f0e9c4425538c107bebb8b06784ab914a8be07b9/plugins/services/src/js/containers/tasks/TaskTable.js#L77`

File: plugins/services/src/js/utils/InstanceUtil.js [hunk 2]
```diff
@@ -38,13 +38,21 @@ const InstanceUtil = {
 
   getNode(task) {
     const nodesList = CompositeState.getNodesList();
-    const id = (task && task.agent && task.agent.id) || null;
+    let id = null;
+
+    if (task.agent && task.agent.id) {
+      id = task.agent.id;
+    } else if (task.agentId) {
```
Comment (weblancaster) [inline plugins/services/src/js/utils/InstanceUtil.js]: no need for `else if`. Seems like we have preferred to use only `if` whenever possible.

**Comment by ahoskins** [comment]
@DanielMSchmidt There is no attached Jira to this work. Please either update the branch to include the Jira or write in the description what this relates to.

File: plugins/services/src/js/utils/InstanceUtil.js (hunk 2 above)
Comment (weblancaster) [reply plugins/services/src/js/utils/InstanceUtil.js]: also, please take a look at @nLight comments here https://github.com/dcos/dcos-ui/pull/2688/files/a4354273ccf7f4f58d4818b8ee7483675243fbcc#diff-419e17a15d4143340f20a92b4aa4fe76

**Comment by DanielMSchmidt** [comment]
@ahoskins Added it 👍

**Comment by DanielMSchmidt** [comment]
Will do the backport once this is merged

**Pull request merged**